src/tools/sfdk/sdkmanager.cpp

```diff
@@ -1348,15 +1349,37 @@ bool SdkManager::mapEnginePaths(QString *program, QStringList *arguments, QStrin
         {cleanSharedSrc, Constants::BUILD_ENGINE_SHARED_SRC_MOUNT_POINT, caseInsensitiveOnWindows}
 #endif
     };
+
     for (const Mapping &mapping : mappings) {
         qCDebug(sfdk) << "Mapping" << mapping.hostPath << "as" << mapping.enginePath;
+        auto mapGreedy = [=](QString string) {
+            QRegularExpression::PatternOption reCaseSensitivity = (mapping.cs == Qt::CaseInsensitive)
+                    ? QRegularExpression::CaseInsensitiveOption : QRegularExpression::NoPatternOption;
```

Comment:
Sorry I missed that before

```suggestion
            QRegularExpression::PatternOption reCaseSensitivity = mapping.cs == Qt::CaseInsensitive
                 ? QRegularExpression::CaseInsensitiveOption
                 : QRegularExpression::NoPatternOption;
```

Reply:
Moved : QRegularExpression:NoPatternOption to separate line. I refuse to remove the parenthesis around the comparison, as that would make the code more difficult to read.

Reply:
Please check `grep -e '^[[:space:]]*?' -C1 -r src/plugins/ ` and the discussion we had before here https://github.com/sailfishos/sailfish-qtcreator/pull/272#pullrequestreview-374961612 (you need to click Show resolved)

Reply:
Fine, just to be consistent with myself, I'll do the same as then: I'll change it, just to get the review accepted. I still think that it is a stupid thing to do. And I probably will fight this every single time.

Reply:
> Fine, just to be consistent with myself, I'll do the same as then: I'll change it, just to get the review accepted. I still think that it is a stupid thing to do. And I probably will fight this every single time.

I hadn't noticed that you had already accepted the review ;) Well, it's changed anyway now..